patches/minecraft/net/minecraft/block/BlockWall.java.patch

```diff
@@ -0,0 +1,51 @@
+--- ../src-base/minecraft/net/minecraft/block/BlockWall.java
++++ ../src-work/minecraft/net/minecraft/block/BlockWall.java
+@@ -141,12 +141,12 @@
+ 
+     public IBlockState func_176221_a(IBlockState p_176221_1_, IBlockAccess p_176221_2_, BlockPos p_176221_3_)
+     {
+-        boolean flag = this.func_176253_e(p_176221_2_, p_176221_3_.func_177978_c());
+-        boolean flag1 = this.func_176253_e(p_176221_2_, p_176221_3_.func_177974_f());
+-        boolean flag2 = this.func_176253_e(p_176221_2_, p_176221_3_.func_177968_d());
+-        boolean flag3 = this.func_176253_e(p_176221_2_, p_176221_3_.func_177976_e());
++        boolean flag = canWallConnectTo(p_176221_2_, p_176221_3_, EnumFacing.NORTH);
++        boolean flag1 = canWallConnectTo(p_176221_2_, p_176221_3_, EnumFacing.EAST);
++        boolean flag2 = canWallConnectTo(p_176221_2_, p_176221_3_, EnumFacing.SOUTH);
++        boolean flag3 = canWallConnectTo(p_176221_2_, p_176221_3_, EnumFacing.WEST);
+         boolean flag4 = flag && !flag1 && flag2 && !flag3 || !flag && flag1 && !flag2 && flag3;
+-        return p_176221_1_.func_177226_a(field_176256_a, Boolean.valueOf(!flag4 || !p_176221_2_.func_175623_d(p_176221_3_.func_177984_a()))).func_177226_a(field_176254_b, Boolean.valueOf(flag)).func_177226_a(field_176257_M, Boolean.valueOf(flag1)).func_177226_a(field_176258_N, Boolean.valueOf(flag2)).func_177226_a(field_176259_O, Boolean.valueOf(flag3));
++        return getWallState(p_176221_1_, p_176221_2_, p_176221_3_, flag, flag1, flag2, flag3, flag4);
```

Comment:
can you replace this line with the vanilla line that was there before and remove `getWallState`?